### ISSUE TITLE
Stop model training of a Neural Net based on certain criteria?

### ISSUE THREAD

**Issue by fleapapa** [01org/daal#13]
Is there any way to stop model training of Neural Net (distributed asynchronous mode) based on certain criteria, for examples, loss function or accuracy of cross verification?

Thanks!

**Comment by Vika-F** [comment]
Please see the answer to #14 .
